src/object.d

```diff
@@ -286,9 +286,12 @@ class TypeInfo
     null if none. */
     @property inout(TypeInfo) next() nothrow pure inout @nogc { return null; }
 
-    /** Return default initializer.  If the type should be initialized to all
-    zeros, an array with a null ptr and a length equal to the type size will be
-    returned. */
+    /**
+     * Return default initializer.  If the type should be initialized to all
+     * zeros, an array with a null ptr and a length equal to the type size will
+     * be returned. For static arrays, this represents a single element of the
```

Comment:
Very minor remark: Is "represents" clear enough? How about "For static arrays, this returns the default initializer for a single element ..."?